Admin Page: adds a notice with some text and a link to the survey page after disconnection.

zinigor opened the pull request:
This is a work in progress which in the end will allow us to ask users to fill out a survey to tell us about the reason why they have disconnected. You are welcome to review and test, some things to consider when you do:
* the notice placement is yet to be determined, if you have a better idea, please leave a comment;
* we may want to display the notice until the user decides to dismiss it rather than just on the redirection page after disconnection.
cc @richardmuscat

dereksmart commented on class.jetpack.php:
```diff
@@ -2366,6 +2366,11 @@ function admin_init() {
 
+		// If the plugin has just been disconnected from WP.com, show the survey notice
+		if ( isset( $_GET['disconnected'] ) && 'true' === $_GET['disconnected'] ) {
+			add_action( 'admin_notices', array( $this, 'disconnect_survey_notice' ) );
```
We have a `jetpack_notices` hook :) It will put the notice in a more appropriate place.  

Great idea BTW

zinigor commented:
@dereksmart thanks for the suggestion to move the notice to `jetpack_notices`, that looks much better! @richardmuscat I have removed one `<br>` tag and fixed the other one to make it self-closing, please take a look at it again. If you need a site running that branch, ping me :)

samhotchkiss commented on class.jetpack.php:
```diff
@@ -2872,6 +2877,26 @@ public static function jetpack_comment_notice() {
+				<div class="squeezer">
+					<h4>
+						<?php echo sprintf(
+							__( 'You have successfully disconnected Jetpack.<br />Would you tell us why? Just <a href="%s">answering two simple questions</a> would help us improve Jetpack.', 'jetpack' ),
```
I feel like this should be split into two strings (on either side of the `<br />`), as we may well have reason to re-use the string "You have successfully disconnected Jetpack.", and it will allow us to perform later styling changes without modifying translations

rase- replied:
Agreed.

roccotripaldi commented:
I tested and reviewed. It is functioning as expected.

Other than the string change mentioned by @samhotchkiss - LGTM

rase- commented:
LGTM to me otherwise than the comment Sam brought up above.

dereksmart commented:
Tested, tried and true!  :shipit:
